### PR TITLE
Add RelativeOrietnationSensor interface

### LINKED ISSUE #15: Define OrientationSensor interfaces
The OrientationSensor interface could define methods and internal slots (quaternion), while subclasses can explicitly define model and what kind of low-level sensors are used for fusion, e.g.:

- AbsoluteOrientationSensor : OrientationSensor, uses gyroscope, acceleromter and magnetometer.
- GeomagneticOrientationSensor : OrientationSensor, uses accelerometer and magnetometer.
- RelativeOrientationSensor : OrientationSensor, uses gyroscope and accelerometer.

### PR DESCRIPTION
**Pull request by alexshalamov**
Partially fixes #15

***
[Preview](https://s3.amazonaws.com/pr-preview/alexshalamov/orientation-sensor/gh-pages.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/orientation-sensor/db8efe5...alexshalamov:9d30aa6.html)

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -242,6 +267,19 @@ orientation sensor's [=latest reading=] would represent 0 (rad) [[SI]] rotation
...
+The RelativeOrientationSensor Model {#relativeorientationsensor-model}
+----------------------------------------------------------------------
+
+The {{RelativeOrientationSensor}} class is a subclass of {{OrientationSensor}} which represents the [=Relative
+Orientation Sensor=].
+
+For the relative orientation sensor the value of [=latest reading=]["quaternion"] represents the rotation of a
+device hosting motion sensors in relation to a stationary reference coordinate system.
+
+Note: The relative orientation sensor data could be more accurate than the one provided by absolute orientation sensor,
```
Comment (kenchris) [inline index.bs]: Breaking the lines around 100 or so would make these fit better in the GitHub review tool :)

Comment (kenchris) [inline index.bs]: Ie the length you use here seems to be the max github allows without scrolling

File: index.bs [hunk 2]
```diff
@@ -242,6 +267,19 @@ orientation sensor's [=latest reading=] would represent 0 (rad) [[SI]] rotation
...
+as the sensor is not affected by magnetic fields. However, the stationary reference coordinate system may drift due to
+the bias introduced by gyroscope sensor.
```
Comment (kenchris) [inline index.bs]: by THE

**Comment by kenchris** [comment]
LGTM, one minor nit

File: index.bs [hunk 3]
```diff
@@ -242,6 +267,19 @@ orientation sensor's [=latest reading=] would represent 0 (rad) [[SI]] rotation
...
 <img src="images/absolute_orientation_sensor_coordinate_system.png" srcset="images/absolute_orientation_sensor_coordinate_system.svg" style="display: block;margin: auto;" alt="AbsoluteOrientationSensor coordinate system.">
 
+The RelativeOrientationSensor Model {#relativeorientationsensor-model}
+----------------------------------------------------------------------
+
+The {{RelativeOrientationSensor}} class is a subclass of {{OrientationSensor}} which represents the [=Relative
+Orientation Sensor=].
+
+For the relative orientation sensor the value of [=latest reading=]["quaternion"] represents the rotation of a
+device hosting motion sensors in relation to a stationary reference coordinate system.
```
Comment (pozdnyakov) [inline index.bs]: Could you add a couple of word describing "stationary reference coordinate system"?

Comment (anssiko) [inline index.bs]: +1

Make the "stationary reference coordinate system" a `<dfn>` similarly to Earth’s reference coordinate system in https://w3c.github.io/orientation-sensor/#earths-reference-coordinate-system

Comment (anssiko) [inline index.bs]: `<dfn>stationary reference coordinate system</dfn> defined as` ...

File: index.bs (hunk 1 above)
Comment (alexshalamov) [inline index.bs]: done

File: index.bs (hunk 3 above)
Comment (alexshalamov) [inline index.bs]: done

Comment (alexshalamov) [inline index.bs]: done

File: index.bs [hunk 4]
```diff
@@ -242,6 +268,24 @@ orientation sensor's [=latest reading=] would represent 0 (rad) [[SI]] rotation
...
+
+The {{RelativeOrientationSensor}} class is a subclass of {{OrientationSensor}} which represents the
+[=Relative Orientation Sensor=].
+
+For the relative orientation sensor the value of [=latest reading=]["quaternion"] represents the
+rotation of a device hosting motion sensors in relation to a [=stationary reference coordinate
+system=]. The [=stationary reference coordinate system=] may drift due to the bias introduced by
+the gyroscope sensor, thus, the rotation value provided by the sensor, may drift over time.
+
+The <dfn>stationary reference coordinate system</dfn> defined as an inertial three dimensional
```
Comment (anssiko) [inline index.bs]: is defined as

Comment (alexshalamov) [inline index.bs]: done

**Comment by alexshalamov** [comment]
@anssiko I added missing 'is', do you have more comments?

**Comment by anssiko** [comment]
LGTM, thanks!

**Pull request merged**